crates/cargo-platform/src/cfg.rs

```diff
@@ -15,13 +16,33 @@ pub enum CfgExpr {
     False,
 }
 
+// Major version restricted to `1`.
+#[derive(Eq, PartialEq, Hash, Ord, PartialOrd, Clone, Debug)]
+pub struct CfgRustVersion {
+    pub minor: u64,
+    pub patch: u64,
+}
+
+impl CfgRustVersion {
+    fn parse(version: &str) -> Option<Self> {
+        let minor_patch = version.strip_prefix("1.")?;
+        let (minor, patch) = match minor_patch.split_once('.') {
+            Some((minor, patch)) => (minor.parse().ok()?, patch.parse().ok()?),
+            None => (minor_patch.parse().ok()?, 0),
+        };
+        Some(Self { minor, patch })
+    }
+}
+
 /// A cfg value.
 #[derive(Eq, PartialEq, Hash, Ord, PartialOrd, Clone, Debug)]
 pub enum Cfg {
```

Comment:
Should we make it non-exhaustive so we don't need to bump version in the future?

Reply:
Keeping it this way makes it easier to find places to update, so unless we have a strong reason for avoiding major bumps, I'd lean towards this (similar with `cargo-util-schemas`)